samples/Meeting/src/app/App.tsx

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { GroupCallLocator, GroupLocator, TeamsMeetingLinkLocator } from '@azure/communication-calling';
+import { CommunicationUserIdentifier } from '@azure/communication-common';
+import { initializeIcons, Spinner } from '@fluentui/react';
+import React, { useEffect, useState } from 'react';
+import {
+  buildTime,
+  callingSDKVersion,
+  createGroupId,
+  fetchTokenResponse,
+  getGroupIdFromUrl,
+  getTeamsLinkFromUrl,
+  isMobileSession,
+  isOnIphoneAndNotSafari,
+  isSmallScreen,
+  navigateToHomePage
+} from './utils/AppUtils';
+import { CallError } from './views/CallError';
+import { MeetingScreen } from './views/MeetingScreen';
+import { EndCall } from './views/EndCall';
+import { HomeScreen } from './views/HomeScreen';
+import { UnsupportedBrowserPage } from './views/UnsupportedBrowserPage';
+import { getThreadId } from './utils/getThreadId';
+import { createThread } from './utils/createThread';
+import { checkThreadValid } from './utils/checkThreadValid';
+import { getEndpointUrl } from './utils/getEndpointUrl';
+import { joinThread } from './utils/joinThread';
+
+console.log(
+  `ACS sample Meeting app. Last Updated ${buildTime} Using @azure/communication-calling:${callingSDKVersion}`
```

Comment:
Can we also mention the chat sdk version used here as meeting composite uses both (should be injectable by webpack the same way `callingSDKVersion` is)

Reply:
okay will add that